data-pipeline/README.md

```diff
@@ -19,9 +19,13 @@ If you'd like to try it out, this section has the setup instructions.
 
 **Prerequisites**
 
-The setup instructions below use [PDM](https://pdm-project.org/) to install Python, manage a Python virtual environment, and manage dependencies. However, PDM is only absolutely necessary for managing dependencies (because the lockfile is in PDM format), so other tools for Python installs and environments will work as well.
+Before setting up the project:
+
+- Your machine will need to have an installed version of Python that meets the `requires-python` constraint in [pyproject.toml](pyproject.toml)
+- That version of Python should be set as your current Python interpreter (if you don't already have a preferred Python workflow, [pyenv](https://github.com/pyenv/pyenv) is good tool for managing Python versions on your local machine).
+
+In addition, if you're planning to make code changes that require adding or removing project dependencies, you'll need `pip-tools` installed on your machine. ([`pipx`](https://github.com/pypa/pipx) is a handy way to install python packages in a way that makes them available all the time, regardless of whatever virtual environment is currently activated.)
```

Comment:
Pip tools install instructions indicate that it should be installed within an activated virtual environment, suggestion here is to signpost that we will deal with that later in these instructions.
```suggestion
In addition, if you're planning to make code changes that require adding or removing project dependencies, you'll need `pip-tools` installed on your machine; we give installation instructions for `pip-tools` below. ([`pipx`](https://github.com/pypa/pipx) is a handy way to install python packages in a way that makes them available all the time, regardless of whatever virtual environment is currently activated.)
```

Reply:
I love your code reviews @elray1 -- not everyone would take the time to review pip-tools documentation as part of this 🙏 

This is an interesting point...I actually don't install `pip-tools` into my project virtual environments. I install it via `pipx`, which makes it available everywhere.

If that's one too many tools to toss around, would it be easier just to add `pip-tools` as a dev dependency here?

[I'm gonna merge this PR, so I can proceed, but can address this point in the next one]

Reply:
Adding pip-tools as a dev dependency would be helpful to me -- I definitely feel like I'm swimming in a sea of tools right now 😅. I may feel better/differently about this in a couple of months when i have workflows more sorted out for myself though.